### DISCUSSION TITLE
Would GitHub add a Lists for repositories?

### DISCUSSION THREAD

**Discussion by winterant** [community/community#15109] [Other Feature Feedback, Questions, & Ideas]
Would GitHub add a Lists for repositories? Just like new stars lists which I can create lists by myself and then take each repositorie to the corresponding list.

In the other word, I want each repository has a category. When I want to find my a repository I can find it in my lists.

**Comment by AndrewScacchi** [comment]
Yeah, same here, kinda feel pointless to star each repo to put into a list.. Just let me put them into list xD xD
